package.json

```diff
@@ -58,7 +58,8 @@
 		"@wordpress/block-editor": "^5.3.0",
 		"@wordpress/blocks": "^8.0.0",
 		"classnames": "^2.3.1",
-		"fast-average-color": "^6.4.0",
+		"fast-average-color": "^6.3.0",
```

Comment:
bump?

Reply:
There's a rebase needed. Was about to do it